### ISSUE TITLE
Feature Request - Home assistant Integration

### ISSUE THREAD

**Issue by dapuzz** [NeiroNx/python-dvr#23]
Hi NeiroN, I really love your implementation and I wonder if you can work on an Home Assistant Integration such as ones listed here:
https://www.home-assistant.io/integrations/#camera

Let me know what do you think. 
Thank you
Dario

**Comment by dave-code-ruiz** [comment]
+1 wait for develope

**Comment by Sergii1991** [comment]
Morning everyone, it would be awesome if we make this issue real. I'm trying to connect somehow two chinese xmeye/ICSee cameras into Home Assistant but after a few days of reading/testing I decline :( 
Is there any option to integrate them?

**Comment by widgetii** [comment]
Yes, just write a code and make a PR

**Comment by dbuezas** [comment]
You can use go2rtc, search the readme and issues for dvrip and icsee

**Comment by Vinndimon** [comment]
> You can use go2rtc, search the readme and issues for dvrip and icsee

As far as I can see, go2rtc does not have an alarm server.

**Comment by dbuezas** [comment]
That's true, but you can at least get the video streams (even for cams without rtsp) and ptz control.

Funny you mention the alarm, I am trying to add alarm support to [this](https://github.com/dbuezas/icsee-ptz), but `setAlarm` doesn't seem to work in this package.

Not all my cameras have the option to add an alarm server so I want to use `setAlarm` and `alarmStart`.

I know the camera is triggering alarms because I can see them with this app https://xmeye.org/xmeye-for-pc/

**Comment by Vinndimon** [comment]
@dbuezas I do that for more than a year now. But I do PTZ controls via python-dvr and some scripts. It is all off-topic because this feature request us about an alarm server.

**Comment by dbuezas** [comment]
You may have mixed it with #45

**Comment by dbuezas** [comment]
On the topic of alarms: the normal ones are broken in master, the fix is #47

**Comment by widgetii** [comment]
Thank you, merged

**Comment by dbuezas** [comment]
I think this issue may be solved now:
* https://github.com/dbuezas/icsee-ptz provides ptz, time setting and motion alarms (using this library)
* https://github.com/AlexxIT/WebRTC provides streaming
